tests/components/tessie/snapshots/test_device_tracker.ambr

```diff
@@ -95,3 +95,99 @@
     'state': 'not_home',
   })
 # ---
+# name: test_device_tracker[device_tracker.test_location-entry]
```

Comment:
Why are the device tracker snapshots changed?

Reply:
I know why, there was a bug which was fixed since this PR was raised. I'll get this removed.

Reply:
This has been removed now.